src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,32 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.0.0
+September 1, 2022
+
+### Removal Notice
+- Kubernetes [1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md) is no longer supprted.
+
+### Deprecation Notice
+- Support for Kubernetes [1.20](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md) has been derprecated and will be removed next release.
+
+### Features
+- Okteto UI now allows users to specify the manifest path when launching a dev environment.
+
+### Improvements
+- Chart has been renamed from "Okteto Enterprise" to "Okteto". This will result in all kubernetes resources being renamed in the okteto namespace.
```

Comment:
Provecho mentioned that this can cause a downtime due to the IP of the load balancer changing when it's recreated. Where should we mention that?